workflow-examples/src/main/java/com/redhat/parodos/examples/simple/task/LoggingWorkFlowTask.java

```diff
@@ -57,7 +57,7 @@ public WorkReport execute(WorkContext workContext) {
 		catch (Exception e) {
 			log.error("There was an issue with the task {}: {}", getName(), e.getMessage());
 		}
-		return new DefaultWorkReport(WorkStatus.COMPLETED, workContext);
+		return new DefaultWorkReport(WorkStatus.FAILED, workContext);
```

Comment:
Why does the LoggingWorkflowTask need to fail always?

Reply:
it should fail after exception is caught

Reply:
Oh right! Thank you for the clarification!